pyiron_atomistics/lammps/potentials.py

```diff
@@ -0,0 +1,414 @@
+                        self.counter,
+                        self.pair_coeff,
+                    )
+                ]
+
+            @property
+            def interacting_species(self) -> list:
+                """
+                Species in LAMMPS notation (i.e. in numbers instead of chemical
+                symbols)
+                """
+                s_dict = dict(
+                    zip(self._species, (np.arange(len(self._species)) + 1).astype(str))
+                )
+                s_dict.update({"*": "*"})
+                return [
+                    " ".join([s_dict[cc] for cc in c])
+                    for c in self._interacting_species
+                ]
+
+            @property
+            def pair_coeff(self) -> list:
+                """
+                Args for pair_coeff. Elements defined in EAM files are
+                complemented with the ones defined in other potentials in the
+                case of hybrid (filled with NULL)
+                """
+                if not self.is_hybrid:
+                    return self._pair_coeff
+                results = []
+                for pc, ps in zip(self._pair_coeff, self._preset_species):
+                    if len(ps) > 0 and "eam" in pc:
+                        s = " ".join(ps + (len(self._species) - len(ps)) * ["NULL"])
+                        pc = pc.replace(" ".join(ps), s)
+                    results.append(pc)
+                return results
+
+        return PairCoeff(
+            is_hybrid="hybrid" in self.pair_style,
+            pair_style=self.df.pair_style,
+            interacting_species=self.df.interacting_species,
+            pair_coeff=self.df.pair_coeff,
+            species=self.species,
+            preset_species=self.df.preset_species,
+        ).results
+
+    @property
+    def pyiron_df(self):
+        """df used in pyiron potential"""
+        return pd.DataFrame(
+            {
+                "Config": [[self.pair_style] + self.pair_coeff],
+                "Filename": [self.filename],
+                "Model": [self.model],
+                "Name": [self.name],
+                "Species": [self.species],
+                "Citations": [self.citations],
+            }
+        )
+
+    def __repr__(self):
+        return self.df.__repr__()
+
+    def _repr_html_(self):
+        return self.df._repr_html_()
+
+    def set_df(self, df):
+        for key in [
+            "pair_style",
+            "interacting_species",
+            "pair_coeff",
+            "preset_species",
+        ]:
+            if key not in df:
+                raise ValueError(f"{key} missing")
+        self._df = df
+
+    @property
+    def df(self):
+        """DataFrame containing all info for each pairwise interactions"""
+        return self._df
+
+    def get_df(self, default_scale=None):
+        if default_scale is None or "scale" in self.df:
+            return self.df.copy()
+        df = self.df.copy()
+        df["scale"] = 1
```

Comment:
The `default_scale` arg is not used unless it has its default value... Was this the intent?:

```suggestion
        df["scale"] = default_scale
```

Reply:
Yeah thanks!